munge/srseng/toimportjson.ts

```diff
@@ -0,0 +1,99 @@
+import { execIfMain } from "execifmain";
+import { writeFile } from "fs/promises";
+import { Command } from "commander";
+import { join as joinPath, resolve as resolvePath } from "path";
+import { loadTsvFile } from "./util";
+import { Dictionary } from "./dictionary";
+import { Transducer } from "hfstol";
+
+const RESOURCE_DIR = resolvePath(
+  __dirname,
```

Comment:
Since this is an ES module wouldn't this be undefined?

Reply:
I did get this working as an ES module, but the level of complexity and brittleness involved in doing so made me decide it wasn’t worth it, and I went back to having TypeScript emit CommonJS instead.